test/common/index.mjs

```diff
@@ -73,6 +73,15 @@ export function leakedGlobals() {
     knownGlobals.push(Symbol);
   }
 
+  // WHATWG APIs
+  if (global.TextEncoder) {
+    knownGlobals.push(TextEncoder);
+  }
+
+  if (global.TextDecoder) {
+    knownGlobals.push(TextDecoder)
+  }
```

Comment:
If these values are added to the global, they should not be checked for. Instead, they should be added to `let knownGlobals = [ ...` on top. And I guess `common/index.js` has to be changed as well?

Reply:
I agree, they should not be checked for conditionally. So I'm gonna add `TextEncoder` and `TextDecoder` to `knownGlobals` in both `test/common/index.mjs` and `test/common/index.js`.

Thanks for the feedback, @BridgeAR!

On another note, I think the recently added `URL` global should be added to both arrays, too (but I guess in another PR).

BTW, the globals are only added in the following case:

```js
const browserGlobals = !process._noBrowserGlobals;
if (browserGlobals) {
  // ...
}
```

[https://github.com/MarkTiedemann/node/blob/5820e161214ba315ab70a1239063bf90f090d6e7/lib/internal/bootstrap/node.js#L73-L74](https://github.com/MarkTiedemann/node/blob/5820e161214ba315ab70a1239063bf90f090d6e7/lib/internal/bootstrap/node.js#L73-L74)

I could not find any documentation regarding this, except that there was once a `--no-browser-globals` flag, which doesn't seem to work anymore.

```
λ node --no-browser-globals
node: bad option: --no-browser-globals
```

So I assume I could remove these 2 lines, too?